### DISCUSSION TITLE
osu wount start

### DISCUSSION THREAD

**Discussion by nusswithoutthedeku** [ppy/osu#15546] [Q&A] [answered]
i wanted to install osu lazer and it doesnt launch

**Comment by bdach** [comment, marked as answer]
please read https://github.com/ppy/osu/discussions/15188 to learn how to report problems properly in a way that we can actually respond to, and attach the requested logs. "it doesn't launch" tells us precisely nothing.

**Reply by frenzibyte** [reply]
Marking as answered due to inactivity.
